app/Models/Post.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace App\Models;
+
+use App\Config\App;
+
+class Post
+{
+    public $db;
+
+    public function __construct()
+    {
+        $this->db = App::db();
+    }
+
+    public function getAllBlogPosts()
+    {
+        $postStatement = $this->db->query('SELECT *,
+                                                    posts.id as postId,
+                                                    users.id as userId,
+                                                    posts.created_at as postCreated,
+                                                    users.created_at as userCreated
+                                                    FROM posts
+                                                    INNER JOIN users
+                                                    ON posts.user_id = users.id
+                                                    ORDER BY posts.created_at DESC');
+        $results = $postStatement->fetchAll();
+        foreach ($results as &$result) {
+            $result['postComments'] = $this->numberofComments($result['postId']);
+        }
+        return $results;
+    }
+
+    public function numberofComments($postId)
+    {
+        $statement = $this->db->prepare('SELECT COUNT(post_id) as postComments from comments WHERE post_id = ?;');
+        $statement->bindParam(1, $postId);
+        $statement->execute();
+
+        return $statement->fetch();
+    }
+
+
+    public function adminBlogInfoHome()
+    {
+        $postInfo = $this->db->query('SELECT *,
+       posts.id as postId,
+       users.id as userId,
+       posts.created_at as postCreated,
+       users.created_at as userCreated
+FROM posts
+         INNER JOIN users
+                    ON posts.user_id = users.id
+ORDER BY posts.created_at DESC;');
+        $results = $postInfo->fetchAll();
```

Comment:
Unnecessary local variable `$result`. The variable can be inlined : `return $postInfo->fetchAll();`